todo_app/data/storage_trello.py

```diff
@@ -0,0 +1,117 @@
+""" Persistence layer, trello edition
+"""
+import os 
+
+from flask import session
+import dotenv
+import requests
+
+from .TrelloSession import TrelloSession
+from .. import trello_config
+
+dotenv.load_dotenv()
```

Comment:
You should find that this is unnecessary - the `python-dotenv` library should automatically load the `.env` file when flask starts up (and you may find it actively unhelpful later, when we want tests to explicitly load in an alternative set of values!)

Reply:
yuck!  I'm inclined to sack off dotenv on this basis!